test/powershell/Language/Scripting/NativeExecution/NativeCommandProcessor.Tests.ps1

```diff
@@ -1,22 +1,32 @@
+$testroot = resolve-path (join-path $psscriptroot ../../..)
+$common = join-path $testroot Common
+$helperModule = join-path $common Test.Helpers.psm1
+
 Describe 'native commands lifecycle' -tags 'Feature' {
 
     BeforeAll {
+        Import-Module $helperModule -force
```

Comment:
Agree, just copied it from the debugger script.
I will make a commit to remove it from both places.